modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -58,12 +58,15 @@ def _unwrap_partitions():
             [p.drain_call_queue() for p in modin_frame._partitions.flatten()]
             if get_ip:
                 return [
-                    [(partition._ip_cache, partition._data) for partition in row]
+                    [
```

Comment:
Could you add an assert that each `list_of_blocks` has length 1? This method already checks below that we are using one of the block partition classes, which should only have one block, but I think it's safer to add an extra check.

Reply:
The assertion was added